src/webgl/p5.RendererGL.js

```diff
@@ -602,6 +606,39 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
     this._curShader = undefined;
   }
 
+  /**
+   * Starts creating a new p5.Geometry.
+   */
+  beginGeometry() {
```

Comment:
Can we update the inline docs here with the same text as in the GeometryBuilder file?